data/blog/2024/2024-09-26-using-laminas-continuous-delivery-and-deployment.md

```diff
@@ -0,0 +1,61 @@
+---
+id: 2024-09-26-using-laminas-continuous-delivery-and-deployment
+author: julian
+title: 'Using Laminas Continuous Delivery and Deployment'
+draft: false
+public: true
+created: '2024-09-26T11:00:00-01:00'
+updated: '2024-09-26T11:00:00-01:00'
+tags:
+  - continuous integration
+---
+
+We detailed the meaning and some inner workings of Continuous Integration (**CI**) in a [previous article](https://getlaminas.org/blog/2024-08-05-using-laminas-continuous-integration.md.html), so here we will focus on what happens after CI completes successfully.
+
+CI is followed by **Continuous Delivery** (**CD**) which gets the code **ready for deployment to the production or staging environment**.
+This ensures that the code is ready to be released **manually**.
+**Continuous Deployment** goes one step further and **automates the release** itself.
```

Comment:
Do we really need all that highlighting? At the moment you can't see any of it anyway!